Microsoft.Toolkit.Uwp/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
```

Comment:
@deltakosh No - it is not required at all here. I will change this in next update.